src/compas/files/obj.py

```diff
@@ -371,69 +448,90 @@ def parse(self):
         self.polylines = [[index_index[index] for index in line] for line in self.reader.lines if len(line) > 2]
         self.faces = [[index_index[index] for index in face] for face in self.reader.faces]
         self.groups = self.reader.groups
+        self.objects = {}
```

Comment:
Just curious why is here not using `defaultdict`  like the reader class?

Reply:
there is no need because i am just parsing the corresponding dict of the reader. so there is no possibility of uninitialised items in the dict

Reply:
I see!